src/client.js

```diff
@@ -0,0 +1,9 @@
+import { is } from 'ramda'
+
+export function browserLanguage (navigatorLanguage = 'cs') {
```

Comment:
This gives out mixed signals: you want to fall back to Czech for undefined states, and fall back to English otherwise. This is okay, but you need to document it.
Add comment to the function, which explains this behaviour. !resolve